docs/CONFIGURING_GLOBAL_HPKE_KEYS.md

```diff
@@ -1,21 +1,83 @@
 # Global HPKE Keypairs
+
 Janus has support for HPKE key advertisement on `/hpke_configs` that are not
-tied to tasks. This is mainly to support the Taskprov extension, but we are
-also [considering having this replace per-task keys entirely][1].
+tied to tasks.
 
 If a global key is configured, requests to `/hpke_configs` that don't provide
 the task ID will provide the list of active global HPKE keys.
 
-This document describes the operational overhead of managing these keys.
+If global keys are to be used exclusively, use the feature flag
+`require_global_hpke_keys`. This will cause Janus to ignore the `?task_id`
+parameter of `/hpke_configs` entirely and serve only global HPKE keys. This
+will become the default behavior in the next major version of Janus.
+
+This document describes the operational considerations of managing global HPKE
+keys.
+
+## Key Rotator
+
+It is strongly recommended to use the `key_rotator` component to manage global
+HPKE key provisioning and lifecycle.
+
+### Binary Key Rotator
+
+Use the `key_rotator` subcommand of the `janus_aggregator` binary to invoke
+the key rotator once. 
+It can be invoked like so:
+```
+export DATASTORE_KEYS=[your database encryption key]
+export PGPASSWORD=[your postgresql password here]
+janus_aggregator key_rotator --config-file [path to configuration yaml]
+```
+
+where the configuration is like so:
+```
+database:
+  url: "postgres://postgres:postgres@localhost:5432/postgres"
+
+key_rotator:
+  hpke: {}
+```
+
+This will create a single key and rotate it through a reasonable lifecycle. See
+the [advanced sample](./samples/advanced_config/key_rotator.yaml) to see
+defaults and what options are available.
+
+Run this command on a cronjob, using your cron scheduling daemon of choice
+(e.g. normal `cron`, systemd timers, Kubernetes CronJobs) on some reasonable
+cadence.
 
-[1]: https://github.com/divviup/janus/issues/1641
+This is the recommended way to run the key rotator.
 
-## Lifecycle
+### In-process Key Rotator
+
+A less recommended, but simpler way to run the key rotator is to run it as part
+of the normal Janus aggregator.
+
+In your aggregator configuration file, write.
+```yaml
+key_rotator:
+  hpke: {}
+```
+
+Then restart the aggregator. This will create a single key and rotate it
+through a reasonable lifecycle. See the
+[advanced sample](./samples/advanced_config/key_rotator.yaml) to see defaults 
```

Comment:
This should point to the aggregator config file sample instead. (that also shows how to configure key_rotator inline)